src/ucs/memory/numa_defs.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_NUMA_DEFS_H_
+#define UCS_NUMA_DEFS_H_
+
+#include <stdint.h>
+
+#define UCS_NUMA_MIN_DISTANCE 10
```

Comment:
why does it need to be in H file?

Reply:
We have references to this module in other files (topo.c)
Need to remove from numa.h (leftovers...)

Reply:
I would expect topo.c to use ucs_topo_default_distance